Multisig/Data/Services/Utils/JSONHTTPClient.swift

```diff
@@ -14,13 +14,15 @@ public protocol JSONRequest: Encodable {
     /// Query parameters
     var query: String? { get }
 
+    var url: String? { get }
```

Comment:
spacing, please. Also please add a documentation comment that explains how this request will be treated differently if the url present vs when it is not present